src/PowerSystems.jl

```diff
@@ -163,12 +165,15 @@ export ReactivePowerDroop
 export ActivePowerDroop
 export ActivePowerPI
 export ReactivePowerPI
+export ActiveRenewableTypeAB
+export ReactiveRenewableTypeAB
 
 # InnerControl Export
 export InnerControl
 export CurrentControl
 export VoltageModeControl
 export CurrentModeControl
+export InnerREECB
```

Comment:
We need a better name here. `REECCurrentControl`

Reply:
recall that REEC means Renewable Energy Electric Controller, so we need at least the B. We could do `RenewableEnergyCurrentControlB` or `RECurrentControlB`.

Reply:
RECurrentControlB is good